src/effects/native/reverbeffect.cpp

```diff
@@ -31,6 +31,7 @@ EffectManifest ReverbEffect::getManifest() {
     time->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
     time->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
     time->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    time->setDefaultLinkType(EffectManifestParameter::LINK_LINKED);
```

Comment:
I disagree on this one. I think the superknob should generally cause a more audible effect on the right and least audible/no effect on the left (except for bipolar superknobs like for filter).